thankyou_letters/models/res_partner.py

```diff
@@ -70,12 +71,14 @@ def _compute_salutation(self):
                     title_salutation + ' ' + partner.firstname
                 partner.informal_salutation = title_salutation + ' ' + \
                     partner.firstname
+                partner.full_salutation = title_salutation + ' ' + \
+                    partner.firstname + ' ' + partner.lastname
             else:
                 partner.salutation = _("Dear friends of ") + \
                     self.env.user.company_id.name
                 partner.short_salutation = partner.salutation
-                partner.informal_salutation = ("Dear friend of ") + \
-                    self.env.user.company_id.name
+                partner.informal_salutation = partner.salutation
```

Comment:
Why did you change the `informal_salutation`? It's not good, it looks like the same, but the informal is in singular `friend` vs `friends`. Please revert this change. The `full_salutation` should be equal to the normal salutation (this is right).